data.tex

```diff
@@ -1,27 +1,71 @@
-% Early Science Data Products
-% Responsible: Leanne Guy
-\section{Early Science Data Products} 
+\section{Early Science Data Products}
 \label{sec:data}
 
-Here we provide a summary of the data products that are expected to be made available as part of the \esp. 
-The definitive source for all LSST data products is the Data Products Definition Document, \citep{LSE-163}.
+All Rubin Observatory LSST data products are produced by the LSST Science Pipelines \cite{2019ASPC..523..521B,2018PASJ...70S...5B}. 
+For an introduction to the Rubin data products, see \citet{RubinDataProductsAbridged}.
+Here we provide a summary of the data products that are expected to be made available as part of the \esp.
+
+Each pre-operations data preview and survey data release will be accompanied by its own release-specific DPDD, giving e.g. the  database schema for the catalogs included in that dataset.
+For an example data release DPDD, see the online DP0.2 documentation.\footnote{\url{https://dp0-2.lsst.io/data-products-dp0-2/}}
+
 The Rubin data rights policy is described in  \cite{RDO-013}.
 
 \subsection{Prompt data products}
 
-Prompt data products are described in detail in the Data Products Definition Document (\DPDD).
-Alert packets are triggered by difference image source detections and transmitted to community alert brokers\footnote{See \url{https://www.lsst.org/scientists/alert-brokers} for the list of selected brokers.} and are publicly available. 
+Data products for transients, variables, and moving objects will be primarily produced by the Prompt Processing pipelines, which will perform reduction, calibration, difference image analysis (DIA), source detection and measurement, and alert distribution within 60 seconds of image readout. 
+They include images, catalogs and alerts. 
+
+Prompt image data products will be made available 80 hours following camera readout. 
```

Comment:
```suggestion
During routine LSST operations, prompt image data products will be made available 80 hours following camera readout. 
```

do we need to say here or elsewhere that images will not be available for commissioning data in less than 30 days?   do we make commissioning PP images available prior to DP1/2?  Tables 3 and 4 suggest not